src/plugins/xref/queries.js

```diff
@@ -0,0 +1,13 @@
+export function getActiveXref(editor, value) {
+  const inline = value.startInline
+  if (!inline) return null
+
+  return inline.type === 'xref' ? inline : null
+}
+
+export function getActiveLink(editor, value) {
+  const inline = value.startInline
+  if (!inline) return null
+
+  return inline.type === 'link' ? inline : null
+}
```

Comment:
Given how trivial those queries are, I'd rather not have them as separate functions.